src/processing/layers/message/from_slack_attachments.js

```diff
@@ -0,0 +1,51 @@
+module.exports = (webhook, matrix) => {
+    if (!webhook.attachments) return;
+    html = "";
+    webhook.format = "html"
+    webhook.attachments.forEach(function (attachment){
```

Comment:
For this project, I prefer to use arrow functions. Eg: `.forEach(attachment => {`